scripts/start_civ.sh

```diff
@@ -71,6 +71,15 @@ GUEST_STATIC_OPTION="\
 
 
 #------------------------------------------------------         Functions       ----------------------------------------------------------
+function check_non_graphical_target() {
+    local target=$(/bin/systemctl get-default)
+    if [[ $target  == *"graphical.target"* ]]; then
+	echo "System is still in graphical.target. Execute below command to set as multi-user target, and reboot before launching GVT-d."
```

Comment:
from the print message it is for gvt-d only, then suggest to move the check inside setup_gvtd function